Add `SPC w n` keybinding for a new window

swt30 opened the pull request:
Seems like a useful thing to include in `+window`. `SPC w n` now creates a new Atom window. I also renamed some uses of the word. Now "window" consistently refers to the entire window, and each window may contain multiple "panes" which can be manipulated from `SPC w ...`

dvcrn commented:
Thank you! Could you squash your commits and change the message to follow the format [described here](https://github.com/dvcrn/proton/blob/master/.github/CONTRIBUTING.md#commit-format)?

swt30 commented:
That's right, I thought I'd seen a CONTRIBUTING.md somewhere. Done!

jessejanderson commented:
Personally I don't really like the renaming of pane to window, it breaks the mnemonic name convention of using `w` as the shortcut.

dvcrn commented:
@jessejanderson can you give some details what you mean?

This PR does the opposite: Currently proton calls internally all the pane navigation and jumping "window" when it should be "pane" instead, for example on the `<SPC> 0-9` keys. (Just uses a hidden flag to hide them)

dvcrn commented on src/cljs/proton/layers/core/keybindings.cljs:
```diff
@@ -142,9 +142,11 @@
+                  :title "new window"}
               :o {:title "close others"
                   :fx panes/close-other-panes}
-              :u {:action "open last-closed window"
```
Oh! Looks like this was a oversight in previous versions as well: `:action` should be `pane:reopen-closed-item` and `:title` should be `open last-closed pane`. Could you update this? 

Also please give it a try if this is actually the right command for re-opening a closed pane.

swt30 replied:
It appears that this command actually opens the most-recently closed tab (buffer) instead. I think it should be moved to  `SPC b u`.

dvcrn commented:
@jessejanderson I agree but the elements are just not windows but panes. Calling them window is not right. 

From the user side, since we hide all of these elements anyway, the user will likely never see any of that pane/window conflict

swt30 commented:
I like to think of it as managing the position of items within the window: splitting it, looking at different parts of it or creating a new window all seem to belong in `SPC w`.

I've moved "open last-closed window" to `SPC b u` and renamed it to "undo buffer destroy", since this is what it actually does.

dvcrn commented:
This makes sense to me. I am going ahead with merging this PR. 

I agree that there is some conflict but having it labeled as window seems unnecessarily confusing. `spc w` is currently doing actions on the current window mainly and not interacting on a set of windows (we don't have that functionality yet). Since a window is made out of panes, the actions related to them fit right in.
